src/backend/bs_worker

```diff
@@ -3133,7 +3133,9 @@ sub dobuild {
   my $bconf = Build::read_config($buildinfo->{'arch'}, "$buildroot/.build.config");
   my $useccache=0;
   if (exists $bconf->{'buildflags:useccache'}) {
-      if (grep {$_ eq "useccache:$packid"} @{$bconf->{'buildflags'} || []}) {
+      my $opackid = $packid;
+      $opackid = $buildinfo->{'releasename'} if ($buildinfo->{'releasename'});
+      if (grep {$_ eq "useccache:$opackid"} @{$bconf->{'buildflags'} || []}) {
```

Comment:
I think this should be `$_ eq "useccache:$opackid" || $_ eq "useccache:$packid"`

Reply:
so do you want to allow putting in both ?
And not check for releasename only in case it is specified ?